### PR TITLE
OKTA-343973: Add LDAP_INTERFACE AuthType to policy api documentation

### PR DESCRIPTION
**Pull request by johnhuang-okta**
## Description:
- **What's changed?** 
  * Add a new AuthType: LDAP_INTERFACE to policy api documentation
- **Is this PR related to a Monolith release?** 
2021.03.0. (confirm with John Huang)

### Resolves:

* OKTA-343973

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/policy/index.md [hunk 1]
```diff
@@ -755,9 +755,9 @@ Specifies a set of Groups whose Users are to be included or excluded
 
 Specifies an authentication entry point
 
-| Parameter | Description | Data Type         | Required | Default |
-| --------- | ----------- | ----------------- | -------- | ------- |
-| authType  |             | `ANY` or `RADIUS` | No       |         |
+| Parameter | Description | Data Type                             | Required | Default |
+| --------- | ----------- | ------------------------------------- | -------- | ------- |
+| authType  |             | `ANY` or `RADIUS` or `LDAP_INTERFACE` | No       |         |
```
Comment (chenningzhang-okta) [inline packages/@okta/vuepress-site/docs/reference/api/policy/index.md:760]: FYI we usually add `<ApiLifecycle access="ea" />` to annotate a specific feature's current lifecycle. For example - https://github.com/okta/okta-developer-docs/pull/1459/files#diff-c46ef0d1a519db411ccedeacea1fe583589f59a754380a2aa2de208c1d103dc6R747, which looks like this:
Screen Shot 2020-11-24 at 4 33 08 PM

**Review by donwoods-okta — APPROVED** [review]
I looked at the rest of the page to see how it describes this parameter and it mostly doesn't, so I don't know what any reader would do with this information.  But it's no worse than it was before in that regard.

File: packages/@okta/vuepress-site/docs/reference/api/policy/index.md (hunk 1 above)
Comment (johnhuang-okta) [reply packages/@okta/vuepress-site/docs/reference/api/policy/index.md]: Thanks for telling me this! I added the annotation. Please take a look and see whether it is correct.

**Review by chenningzhang-okta — APPROVED** [review]
PR approved.
However I do feel the text layout might be a little confusing as the lifecycle annotation doesn't seem to be specific to the `LDAP_INTERFACE` option only. A better option might be to remove the annotation from the table and simply add a note below the table stating that the `LDAP_INTERFACE` data type option is in EA lifecycle. Doc team please advise.
Screen Shot 2020-11-25 at 1 14 28 PM

**Comment by jakubvul-okta** [comment]
@chenningzhang-okta I agree that is more clear.

File: packages/@okta/vuepress-site/docs/reference/api/policy/index.md [hunk 2]
```diff
@@ -755,9 +755,11 @@ Specifies a set of Groups whose Users are to be included or excluded
...
 Specifies an authentication entry point
 
-| Parameter | Description | Data Type         | Required | Default |
-| --------- | ----------- | ----------------- | -------- | ------- |
-| authType  |             | `ANY` or `RADIUS` | No       |         |
+| Parameter | Description                              | Data Type                             | Required | Default |
+| --------- | ---------------------------------------- | ------------------------------------- | -------- | ------- |
+| authType  | This tells how the user is authenticated | `ANY` or `RADIUS` or `LDAP_INTERFACE` | No       |  `ANY`  |
+
+`LDAP_INTERFACE` is an <ApiLifecycle access="ea" /> release.
```
Comment (chenningzhang-okta) [inline packages/@okta/vuepress-site/docs/reference/api/policy/index.md:762]: I think this statement is confusing. `LDAP_INTERFACE` is not a release. My suggestion:
Note: The `LDAP_INTERFACE` data type option is an <ApiLifecycle access="ea" /> feature.

also nit: I think it'd be cleaner to put the statement in a note block, something like this:
Screen Shot 2020-12-08 at 11 57 04 AM

File: packages/@okta/vuepress-site/docs/reference/api/policy/index.md [hunk 3]
```diff
@@ -755,9 +755,11 @@ Specifies a set of Groups whose Users are to be included or excluded
 
 Specifies an authentication entry point
 
-| Parameter | Description | Data Type         | Required | Default |
-| --------- | ----------- | ----------------- | -------- | ------- |
-| authType  |             | `ANY` or `RADIUS` | No       |         |
+| Parameter | Description                              | Data Type                             | Required | Default |
+| --------- | ---------------------------------------- | ------------------------------------- | -------- | ------- |
+| authType  | This tells how the user is authenticated | `ANY` or `RADIUS` or `LDAP_INTERFACE` | No       |  `ANY`  |
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/reference/api/policy/index.md:760]:
```suggestion
| `authType`  | Specifies how the user is authenticated | `ANY` or `RADIUS` or `LDAP_INTERFACE` | No       |  `ANY`  |
```

**Pull request merged**